src/features/smartSearch/components/sankeyDiagram/SmartSearchSankeyProvider.tsx

```diff
@@ -0,0 +1,82 @@
+import { createContext, FC, ReactNode, useContext } from 'react';
+
+import makeSankeySegments from './makeSankeySegments';
+import useSmartSearchStats from 'features/smartSearch/hooks/useSmartSearchStats';
+import { ZetkinSmartSearchFilter } from '../types';
+import { SankeyConfig, SankeySegment } from './types';
+
+type SmartSearchSankeyProviderProps = {
+  arrowDepth?: number;
+  arrowWidth?: number;
+  children: ReactNode;
+  color?: string;
+  diagWidth?: number;
+  filters: ZetkinSmartSearchFilter[];
+  hoverColor?: string;
+  margin?: number;
+};
+
+type SankeyContextValue = {
+  config: SankeyConfig;
+  segments: SankeySegment[];
+};
+
+const SankeyContext = createContext<SankeyContextValue | null>(null);
+
+const SmartSearchSankeyProvider: FC<SmartSearchSankeyProviderProps> = ({
+  arrowDepth = 10,
+  arrowWidth = 20,
+  children,
+  color = '#cccccc',
```

Comment:
Theme color

Reply:
I don't care too much about using theme colors here, because this is just a default defined in the column. Using the themes in this context requires importing the `theme` (rather than retrieving it from context with `useTheme()`) which means that it's statically built into the app. That takes away a lot of the benefits of themes with regards to things like "dark mode", but I do agree that it has other benefits, like having a central repository of all "default" (light mode) colors.